lua/nvim-treesitter/install.lua

```diff
@@ -311,7 +311,7 @@ local function do_download_git(repo, project_name, cache_dir, revision, project_
   local r = job.run({
     'git',
     'clone',
-    '--depth=1',
+    '--filter=blob:none',
```

Comment:
Not sure that's an improvement here, as these repos get deleted immediately (unlike plugins, which stay around and may want to checkout around), and `--depth=1` is still a bit faster/smaller in my limited experience?

Reply:
We can't use depth since it won't collect all the reds so the checkout command later can fail.

Reply:
Right, because different branches (I'd assumed we'd clone them directly). Never mind then!